spec/controllers/super_admin/account_users_controller_spec.rb

```diff
@@ -19,4 +19,17 @@
       end
     end
   end
+
+  describe 'DELETE /super_admin/account_users/:id' do
+    context 'when user has only one account' do
+      let(:account_user) { create(:account_user) }
+
+      it 'redirects to the account details' do
+        sign_in super_admin
+        delete "/super_admin/account_users/#{account_user.id}"
+        expect(response).to have_http_status(:redirect)
+        expect(response).to redirect_to("/super_admin/accounts/#{account_user.account.id}")
```

Comment:
Can you check if the user is not present or the user is destroyed after the delete method?

Reply:
@murtaza-bagwala Please notify once you update the comment.

Reply:
sure